### PR TITLE
Remove httpmime dependency and upgraded to httpclient5

### PR DESCRIPTION
**Pull request by sharmakh**
Remove httpmime dependency and upgraded to httpclient5

Resolves https://github.com/IBM/cics-bundle-common/issues/77

### CODE REVIEW

**Comment by davenice** [comment]
This looks good to me. I'll approve, I think @sharmakh had a couple more tests she wants to run before we merge it though, so I'll let @sharmakh merge it when she's ready.

File: pom.xml
Comment (davenice) [inline pom.xml:314]: Actually one minor change - can we go straight to the latest available? Also a quick double check whether we require httpcore5 separately.

**Review by davenice — CHANGES_REQUESTED** [review]

File: pom.xml
Comment (sharmakh) [reply pom.xml]: version is updated...we are using some imports from httpcore which are not available in httpclient5 so we cant remove that

File: pom.xml
Comment (davenice) [inline pom.xml:314]: Could this go to 5.2.4 too? :-D
```suggestion
      <version>5.2.4</version>
```

Comment (sharmakh) [reply pom.xml]: done

**Comment by sharmakh** [comment]
@davenice i have tested it we can go ahead with merging this.

**Review by davenice — APPROVED** [review]

**Pull request merged**
